crates/wasmcloud-host/src/control_interface/ctlactor.rs

```diff
@@ -55,12 +55,12 @@ impl Actor for ControlInterface {
 impl Handler<PublishEvent> for ControlInterface {
     type Result = ResponseActFuture<Self, ()>;
 
-    fn handle(&mut self, msg: PublishEvent, _ctx: &mut Context<Self>) -> Self::Result {
-        /*if self.client.is_none() {
-        //    trace!("Skipping control interface event (interface disabled)");
+    fn handle(&mut self, _msg: PublishEvent, _ctx: &mut Context<Self>) -> Self::Result {
+        if self.client.is_none() {
+            //    trace!("Skipping control interface event (interface disabled)");
             return Box::pin(async move {}.into_actor(self));
         }
-        let evt = msg
+        let evt = _msg
```

Comment:
Is the `_` prefix intentional here on msg? I thought that was usually reserved for denoting that you wouldn't use that variable

Reply:
Hah, this looks like an "IDE refactor". I'll take care of it.